int32[] startPoint
int32[] targetPoint
---
uint32[] pathx
uint32[] pathy
